### DISCUSSION TITLE
Is it possible to know all remote harvesters plots in single farmer

### DISCUSSION THREAD

**Discussion by 21497724** [Chia-Network/chia-blockchain#6046] [General]
I have setup a single farmer and multiple harvesters, I there any why to know how many plots for each connected harvester?
https://github.com/Chia-Network/chia-blockchain/wiki/Farming-on-many-machines

**Comment by lvcivs** [comment]
This is being worked on. I think I saw a pull request for this feature, so it will likely be in one of the next releases.
